src/main/java/lotto/model/LotteryMachine.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import lotto.model.enums.WinningMoney;
+
+public class LotteryMachine {
+
+    private Map<WinningMoney, Integer> store;
+
+    private PersonLotto personLotto;
+
+    private WinningLotto winningLotto;
+
+    public LotteryMachine(PersonLotto personLotto, WinningLotto winningLotto) {
+        this.store = new EnumMap<>(WinningMoney.class);
+        this.personLotto = personLotto;
+        this.winningLotto = winningLotto;
+    }
+
+    public Map<WinningMoney, Integer> drawingLotto(Bonus bonus) {
+        personLotto.getPurchaseLotto().forEach(
+                lotto -> {
+                    int matchCount = checkMatch(lotto);
+                    WinningMoney result = WinningMoney.from(matchCount, checkBonus(lotto, bonus));
```

Comment:
정말 사소하지만, 추출된 지역 변수를 사용할 거면 둘다 지역 변수를 추출, 그렇지 않다면 둘다 인라인으로 구성하는 것이 일관성이 있을 것 같습니다!
개인적으로는 변수 추출을 하는 편이 가독성이 더 좋을 것 같아요
```suggestion
                    boolean hasBonus = checkBonus(lotto, bonus);
                    WinningMoney result = WinningMoney.from(matchCount, hasBonus);
```